### PR TITLE
Receive

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

**Comment by plondon** [comment]
screen shot 2016-03-26 at 10 13 55 am
screen shot 2016-03-26 at 10 14 18 am

File: app/partials/request.jade [hunk 1]
```diff
@@ -8,86 +8,100 @@
...
+            span.help-block(ng-show="requestForm.label.$error.gap", translate="LABEL_ERROR_BIP_44_GAP") 
+
+      div.prl.pll(ng-show="advanced && fields.to.index != undefined")
+        span.ti-arrow-right
+        span.ti-arrow-down.center
+
+      div.payment-url.flex-around(ng-if="paymentRequestURL()" ng-class="{'row': !advanced, 'column': advanced, 'flex-center': advanced}")
+        .flex-column.receive-address.flex-center(ng-class="{'max-width': advanced}"
+                                                 popover-placement="top" 
+                                                 uib-popover="{{copyHelper}}"
```
Comment (Sjors) [inline app/partials/request.jade]: does `uib-popover="{{ 'COPY_HELPER' | translate }}"` not work here?

**Comment by Sjors** [comment]
Can you squash this into a single commit?

**Comment by Sjors** [comment]
utACK

**Pull request merged**